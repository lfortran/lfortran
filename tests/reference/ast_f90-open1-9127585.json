{
    "basename": "ast_f90-open1-9127585",
    "cmd": "lfortran --show-ast-f90 --no-indent --no-color {infile}",
    "infile": "tests/open1.f90",
    "infile_hash": "932467cb6b7bcbf97a0b9a7c90ea1ca8074bde26145accfc0bb252c2",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast_f90-open1-9127585.stdout",
    "stdout_hash": "6d193dc22ef86174299ec315108e34a1ae76e1b5ddcf0f3dbeef4567",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}