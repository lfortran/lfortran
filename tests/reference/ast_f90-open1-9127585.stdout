program open1
! Tests for syntax (AST) only:
open(file="p_cc", newunit=u)
open(task_lun, file=taskfile)
open(unit=gmv_lun, FILE=fname, IOSTAT=ier, POSITION="rewind")
open(newunit=this%lun, file=outfile, status="replace", iostat=ios)
open(unit=10, file="test1.inp", position="rewind", action="read", status="old")
open(UNIT=funit, FILE=tname, STATUS=fstat, ACCESS="SEQUENTIAL", ACTION=faction, IOSTAT=ierr)
end program open1
